spec/src/utils/request-queue.js

```diff
@@ -37,7 +38,7 @@ describe('ConstructorIO - Utils - Request Queue', () => {
     });
 
     it('Should add url requests to the queue', () => {
-      const requests = new RequestQueue();
+      const requests = new RequestQueue(options);
```

Comment:
necessary?

Reply:
yup, since we're instantiating the request queue directly it doesn't have the options that are passed through from the constructor or tracker module

Reply:
It shouldn't need them though since it's defaulted?